twoliter/src/tools.rs

```diff
@@ -122,10 +125,18 @@ async fn test_install_tools() {
     assert!(toolsdir.join("tuftool").is_file());
 
     // Check that the mtimes match.
-    let dockerfile_metadata = fs::metadata(toolsdir.join("Dockerfile")).await.unwrap();
+    let build_dockerfile_metadata = fs::metadata(toolsdir.join("build.Dockerfile"))
+        .await
+        .unwrap();
+    let repack_dockerfile_metadata = fs::metadata(toolsdir.join("repack.Dockerfile"))
+        .await
+        .unwrap();
```

Comment:
nit: you don't need to duplicate this check - all the files should have the same mtime, `Dockerfile` was just standing in for one that would always be present.